src/accordion/Accordion.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  accordionID: ``,
```

Comment:
What if it is a class? ID feels very specific. Maybe something like accordingEl?